src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -637,9 +637,7 @@ private function getJsonSchema(bool $v3, \ArrayObject $definitions, string $reso
         $schema = new Schema($v3 ? Schema::VERSION_OPENAPI : Schema::VERSION_SWAGGER);
         $schema->setDefinitions($definitions);
 
-        $this->jsonSchemaFactory->buildSchema($resourceClass, $format, $type, $operationType, $operationName, $schema, $serializerContext, $forceCollection);
-
-        return $schema;
+        return $this->jsonSchemaFactory->buildSchema($resourceClass, $format, $type, $operationType, $operationName, $schema, $serializerContext, $forceCollection);
```

Comment:
This is a consequence of the fix

Reply:
@soyuka: Refs https://github.com/api-platform/core/issues/4215#issuecomment-824873494. I have added a note to the changelog entry